torch/csrc/jit/pickler.cpp

```diff
@@ -414,6 +468,9 @@ OpCode Unpickler::readInstruction() {
     case OpCode::NEWFALSE: {
       stack_.emplace_back(false);
     } break;
+    case OpCode::NONE: {
```

Comment:
I don't think this is being tested

Reply:
Could you add one test where it isn't typed as Optional? This can happen e.g. for a return value of a function

Reply:
It has to be typed as `Optional`, you can't have an attribute with the type annotation `None`